x-pack/plugins/ml/public/util/chart_utils.js

```diff
@@ -228,6 +228,17 @@ export function getTickValues(startTimeMs, tickInterval, earliest, latest) {
   return tickValues;
 }
 
+export function getXTransform(t) {
```

Comment:
I think this function needs a comment as to what it does and when it needs to be used

Reply:
👍 There were some comments in the code where this was initially not a separate function but inline code. I moved those comments and adapted them to this place.